### DISCUSSION TITLE
How to edit Magic Mirror config?

### DISCUSSION THREAD

**Discussion by Jonisos** [tteck/Proxmox#2108] [Help Needed] [answered]
Hey!
basically I am wondering how I can edit the config.js?

Background: Previously I was using a ZFS filesystem and could use WinSCP to edit the MagicMirror files but I switched to LVM-thin and well, the folders can not be found. I could not find a good tutorial on how I can mount (?) the new block storage so it's accessible in WinSCP. I was also lost with the MagicMirror LXC command line since this does not seem to have access to the "/opt/magicmirror/config/config.js" path. Could you please give me some direction?

**Comment by tteck** [comment, marked as answer]
`nano /opt/magicmirror/config/config.js` didn't work?

**Reply by h3xtr3m** [reply]
and how can i acces the rest of the files system. it is working fine.i just want to add some modules and it´s a bit different for me in comparison to pi. 
i´m just a hobby programmer:D sorry

**Reply by tteck** [reply]
Install File browser

**Comment by Jonisos** [comment]
Ah that worked, wow. Thanks!!
I always tried to use "ls" or "cd" commands but I could not see other folders...

Thank you very much for the fast response!
